### ISSUE TITLE
not working on macos

### ISSUE THREAD

**Issue by norwood867** [smurfix/distkv#1]
Hi, in trying to install distmqtt on macos I found that systemd-python doesn't seem compatible. Not sure if that is expected or not, I just thought I'd report it

**Comment by smurfix** [comment]
Yeah, systemd is Linux specific. If you have some "how to tell macos that a service which it just started is now ready" code I'd be happy to include it.

**Comment by Ninpo** [comment]
> Yeah, systemd is Linux specific. If you have some "how to tell macos that a service which it just started is now ready" code I'd be happy to include it.

Knowing Apple, have you tried returning the 👍 emoji?

**Comment by smurfix** [comment]
Heh. I don't have a Mac, so … no.

**Comment by norwood867** [comment]
👍🏻
